lib/internal/url.js

```diff
@@ -160,15 +141,19 @@ function serializeTupleOrigin(scheme, host, port) {
 // Refs: https://url.spec.whatwg.org/#concept-url
 class URLContext {
   constructor() {
-    this.flags = 0;
-    this.scheme = ':';
+    this.href = '';
```

Comment:
Non-blocking, but a potential good minor cleanup here would be to move this into initializers... like

```js
class URLContext {
  flags = 0;
  scheme = ':';
  href = '';
  //... and so so.
```

Reply:
I agree. Updating pull-request soon.

Reply:
Done 👍